test/build-ecs/fixtures/simple-scene-with-bundled/package.json

```diff
@@ -27,5 +27,11 @@
     "eth-connect",
     "dcl-test-lib-integration",
     "rollup-demo-decentraland-lib"
+  ],
+  "bundleDependencies": [
```

Comment:
why adding this bundleDependencies?

Reply:
mm i think make tests update this for me.